### ISSUE TITLE
[Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30013]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Pages require sensory characteristics to be understood or operable.

### Issue Details
No alternative is provided to screen reader users to inform them that the first `h2` on the page is an alert as the "!" icon is set hidden="true". This is particularly relevant in this context as it may be easy to confuse the alert's heading as being directly related with other parts of the page. Notifying screen reader users that this is an alert and not standard main content should help mitigate that issue.

### VA.gov Experience Standard
[Category Number 11, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.3_A

---

### Platform Recommendation
Using the new va alert web component should fix this. Otherwise, [reference this ticket on sr-only alert alternatives.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19674)

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by joshkimux** [comment]
@Mottie I wasn't able to recreate this issue, both routes I took for confirming or adding/removing dependents led me to either:
- An info alert that started with "Note:" which is sufficient for alt text
- A confirmation alert which does not require alt text for the check mark

Closing this out!
